src/main/java/me/drendov/HouseKeeping/HouseKeeping.java

```diff
@@ -190,5 +197,27 @@ synchronized public String getMessage(Messages messageID, String... args) {
         return message;
     }
 
+    private void checkDependency(String name) {
+        if (this.getServer().getPluginManager().getPlugin(name) == null) {
+            logger.info("Missing dependency '" + name + "'. Cannot continue.");
+            getPluginLoader().disablePlugin(this);
+            return;
+        }
+    }
+
+    public static void checkAbsence() {
+        Essentials ess = (Essentials)Bukkit.getPluginManager().getPlugin("Essentials");
+        UserMap userMap = ess.getUserMap();
+
+        Date date = new Date();
+
+        long now = date.getTime();
+        long minBeforeAbsent = now - (long)HouseKeeping.getInstance().config.getDaysWithholdFunds() * (long)86400000;
 
+        for (UUID uuid : ess.getUserMap().getAllUniqueUsers()) {
+            User player = userMap.getUser(uuid);
+            if (player != null && player.getLastLogin() < minBeforeAbsent)
+                logger.info(String.format("Player %s is absent for %d days (%s$)", player.getName(), (now - player.getLastLogin()) / 86400000, player.getMoney().toString()));
```

Comment:
Please use short version, eg "%s, % days, $%s"
Also, round up a balance to whole numbers